pyramid_oereb/lib/processor.py

```diff
@@ -106,11 +106,45 @@ def is_inside_plr(theme_code):
                 themes_to_move.append(i)
         themes_to_move.reverse()
         for idx in themes_to_move:
-            extract.not_concerned_theme.append(extract.concerned_theme.pop(idx))
+            new_not_concerned_theme = extract.concerned_theme.pop(idx)
+            log.debug("plr_tolerance_check() adding to not_concerned_theme theme: {}"
+                      .format(new_not_concerned_theme)
+                      )
+            extract.not_concerned_theme.append(new_not_concerned_theme)
 
         real_estate.public_law_restrictions = self.get_legend_entries(inside_plrs, outside_plrs)
         return extract
 
+    def order_extract_toc(self, extract):
```

Comment:
just for convenience reason: shouldn't we call it "reorder_extract_toc"

Reply:
Yes, and the name was misleading, too. It is only about the not_concerned part.